internal/driver/driver.go

```diff
@@ -102,10 +110,31 @@ func (s *Driver) HandleReadCommands(deviceName string, protocols map[string]mode
 			}
 
 			rxbuf := hex.EncodeToString(s.generic[deviceLocation].rxbuf)
-			s.lc.Debugf(fmt.Sprintf("Driver.HandleReadCommands(): Received Data =  %s", rxbuf))
+			s.lc.Debugf(fmt.Sprintf("Driver.HandleReadCommands(): Received Data = %v", rxbuf))
```

Comment:
Please use `%s` instead of `%v` as the type of `rxbuf` is explicit to be string. Same comment for other log messages.

Reply:
Addressed.